### PR TITLE
Observation plan error handling

### PR DESCRIPTION
**Pull request by mcoughlin**
This PR adds checks and error handling for observation plans and names.

### CODE REVIEW

File: static/js/ducks/observationPlanNames.js [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+  switch (action.type) {
+    case FETCH_OBSERVATION_PLAN_NAMES_OK: {
+      return action.data;
+    }
+    default:
+      return state;
+  }
+};
+
+store.injectReducer("observationPlanNames", reducer);
```
Comment (guynir42) [inline static/js/ducks/observationPlanNames.js:35]: Can this fit in the same file as the other observation plan reducers?

**Review by guynir42 — APPROVED** [review]
Looks good. One question/request:

File: static/js/ducks/observationPlanNames.js (hunk 1 above)
Comment (mcoughlin) [reply static/js/ducks/observationPlanNames.js:35]: I am moving this to a duck called observationPlans.

Comment (guynir42) [reply static/js/ducks/observationPlanNames.js]: Sounds good. Is this the only observation-plan-related reducer we have?

Comment (mcoughlin) [reply static/js/ducks/observationPlanNames.js]: yes because the observation plan is always tied to a single gcn event before.

Comment (guynir42) [reply static/js/ducks/observationPlanNames.js]: OK. If it makes sense to you, then it makes sense to me.

**Pull request merged**